ci/quickstart-confluence-master-parms.json

```diff
@@ -46,5 +46,9 @@
   {
     "ParameterKey": "ExportPrefix",
     "ParameterValue": "$[taskcat_random-string]"
+  },
+  {
+    "ParameterKey": "CollaborativeEditingMode",
+    "ParameterValue": "synchrony-separate-nodes"
```

Comment:
Our default taskcat plan never deployed the synchrony nodes. Lets do this by default now to ensure we arent missing anything